src/main/cpp/util/strings.cc

```diff
@@ -271,6 +273,13 @@ void StringPrintf(string *str, const char *format, ...) {
   delete[] buf;
 }
 
+std::string Quoted(const std::string s)
+{
+  std::ostringstream ss;
+  ss << std::quoted( s.c_str() );
+  return ss.str();
+}
+
```

Comment:
I'm not sure if there's a better way to achieve quoting that java expects the argfile to have.